mysql-test/suite/rpl/t/rpl_rewrite_db_sys_vars.test

```diff
@@ -0,0 +1,103 @@
+-- source include/have_binlog_format_mixed.inc
+-- source include/master-slave.inc
+
+--echo #
+--echo # MDEV-15530 Variable replicate_rewrite_db cannot be found
+--echo #            in "show global variables"
+--echo #
+
```

Comment:
Dynamic updates to the replication filter should be lost after server restart. Can you add a test case to validate that `replicate_rewrite_db` is reverted to the `my.cnf` specification after restart?

Reply:
Ah, yeah `reset_slave.inc` and `start_slave.inc` only work on the slave threads. To restart the mariadbd process, use
```
--let $rpl_server_number= 2
--source include/rpl_restart_server.inc
```